pkg/datapath/loader/compile.go

```diff
@@ -57,6 +57,12 @@ const (
 	hostEndpointObj          = hostEndpointPrefix + ".o"
 	hostEndpointObjDebug     = hostEndpointPrefix + ".dbg.o"
 	hostEndpointAsm          = hostEndpointPrefix + "." + string(outputAssembly)
+
+	networkPrefix   = "bpf_network"
+	networkProg     = networkPrefix + "." + string(outputSource)
+	networkObj      = networkPrefix + ".o"
```

Comment:
If these are purely for documentation, we can add them as comments to work around the linter. If you intended to use them, then by all means let's do that ;)

Reply:
can be removed